subprojects/core-api/src/main/java/org/gradle/api/file/FileSystemOperations.java

```diff
@@ -34,6 +36,38 @@
 @ServiceScope(Scopes.Build.class)
 public interface FileSystemOperations {
 
+    /**
+     * Creates a {@link CopySpec} which can later be used to copy files or create an archive. The given closure is used
+     * to configure the {@link CopySpec} before it is returned by this method.
+     *
+     * @param closure Closure to configure the CopySpec
+     * @return The CopySpec
+     * @since 8.5
+     */
+    @Incubating
+    CopySpec copySpec(@DelegatesTo(CopySpec.class) Closure closure);
```

Comment:
❌ Let's not add the Closure taking method. It is not necessary because runtime decoration will add this for Groovy consumers automatically.

Reply:
Removed, the overload that takes an `Action` needs to stay?

Reply:
Yes!